### ISSUE TITLE
calculate_methylation_frequency crashes

### ISSUE THREAD

**Issue by scoa-git** [jts/nanopolish#186]
Hi!

We successfully (?) called methylation using 
` /home/qf0297/nanopolish/nanopolish call-methylation -t 8 -r mtdna_reads_alba.fasta -g rCRS_ME.fasta -b sorted_mtdna_alba.bam > methylation.tsv`

This seemed to work fine, producing a large output file. When running 
`python ~/nanopolish/scripts/calculate_methylation_frequency.py -c 2.5 -i methylation.tsv > meth_frequencies.tsv`

the terminal reports
```
 # error in calculate_methylation_frequency.py
qf0297@qf0297-ESPRIMO-P420:/media/qf0297/Volume/data_np/20170627_mtDNA_HEK/alba/fast5_alba$ python /home/qf0297/nanopolish/scripts/calculate_methylation_frequency.py -c 2.5 -i methylation.tsv > meth_frequencies.tsv
  File "/home/qf0297/nanopolish/scripts/calculate_methylation_frequency.py", line 76
    print "\t".join(["chromosome", "start", "end", "num_cpgs_in_group", "called_sites", "called_sites_methylated", "methylated_frequency", "group_sequence"])
             ^
SyntaxError: invalid syntax
```

How can this be resolved? Is this an issue in the output? The methylation.tsv looks quite plausible. 

Thank you!
Stefan

**Comment by jts** [comment]
What version of python are you using?

**Comment by scoa-git** [comment]
Python 3.6.1

**Comment by jts** [comment]
Ok - I'll change that script to support python3-style prints. In the meantime it should work if you use python2

**Comment by jts** [comment]
Fixed in 1ccd559

**Comment by scoa-git** [comment]
Hi!

I cloned nanopolish again, but it still gives the same error. 

`qf0297@qf0297-ESPRIMO-P420:/media/qf0297/Volume/data_np/20170627_mtDNA_HEK/alba/fast5_alba$ python ~/nanopolish/scripts/calculate_methylation_frequency.py -c 2.5 -i methylation.tsv > meth_frequencies.tsv
  File "/home/qf0297/nanopolish/scripts/calculate_methylation_frequency.py", line 82
    print "\t".join([str(x) for x in [c, s, e, sites[key].group_size, sites[key].called_sites, sites[key].called_sites_methylated, f, sites[key].sequence]])
             ^
SyntaxError: invalid syntax
qf0297@qf0297-ESPRIMO-P420:/media/qf0297/Volume/data_np/20170627_mtDNA_HEK/alba/fast5_alba$ `

But the script is however indeed the new one.

`cat calculate_methylation_frequency.py`
`[...]`
```
print("\t".join(["chromosome", "start", "end", "num_cpgs_in_group", "called_sites", "called_sites_methylated", "methylated_frequency", "group_sequence"]))

for key in sites:
    if sites[key].called_sites > 0:
        (c, s, e) = key.split(":")
        f = float(sites[key].called_sites_methylated) / sites[key].called_sites
        print "\t".join([str(x) for x in [c, s, e, sites[key].group_size, sites[key].called_sites, sites[key].called_sites_methylated, f, sites[key].sequence]])
```

The new nanopolish also gives this error when opening the help
`qf0297@qf0297-ESPRIMO-P420:/media/qf0297/Volume/data_np/20170627_mtDNA_HEK/alba/fast5_alba$ python ~/nanopolish/nanopolish --help
  File "/home/qf0297/nanopolish/nanopolish", line 1
SyntaxError: Non-UTF-8 code starting with '\xc0' in file /home/qf0297/nanopolish/nanopolish on line 2, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
`

`qf0297@qf0297-ESPRIMO-P420:/media/qf0297/Volume/data_np/20170627_mtDNA_HEK/alba/fast5_alba$ python ~/nanopolish/nanopolish variants
  File "/home/qf0297/nanopolish/nanopolish", line 1
SyntaxError: Non-UTF-8 code starting with '\xc0' in file /home/qf0297/nanopolish/nanopolish on line 2, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
`
